website/docs/core-concepts/building-ui/dynamic-ui/README.md

```diff
@@ -46,3 +46,9 @@ Get Category Dropdown (Default Option property)
 Here Table1 is the name of the widget
 
 ![Click to expand](</img/form_-_table.gif>)
+
+## [Height](./../../../reference/widgets/README.md#height)
+The Height property describes how a widget’s height reacts to content changes. This is a configurable property. The configuration to this property can be found in the property pane under the section `General`, with the property name `Height`. It has three possible configurations:
+- [Fixed](./../../../reference/widgets/README.md#fixed)
```

Comment:
Please add an absolute path. Also, remove the .md extension. Please add the URL followed by the anchor. `/reference/widgets/#fixed`